pact/genesis/devnet/development/ns.yaml

```diff
@@ -0,0 +1,23 @@
+codeFile: ../../../namespaces/ns.pact
```

Comment:
Let's swap this ns.yaml (and other devnet ones) with what's in testnet, and not duplicate the devnet ones.

Thus
- devnet/development and devnet/fasttimedCPM share the "short" ns.yaml currently in testnet
- testnet/ns.yaml uses this file

Reply:
Fixed